app/controllers/projects_controller.rb

```diff
@@ -576,7 +578,7 @@ def set_project
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def project_params
-    result = params.require(:project).permit(:name, :public_access, user_ids: [])
+    result = params.require(:project).permit(:name, :public_access, :description, user_ids: [])
     result[:name] = sanitize(result[:name]) if result[:name]
```

Comment:
Before merging, we need to check with @davidrissato what we should do for sanitization of the “description” input (for security)